File: Looping/Loop inside loop/Other Lang/C/LoopInLoop5.c
#include <stdio.h>

void main() {
	for (int vert = 1; vert <= 8; vert++) {
		//Nested Loops Gave Me Anxiety...
		for (int horz = vert; horz <= 8; horz++) {
			printf("%d ", horz);
		}
		printf("\n");
	}
}

File: Looping/Loop inside loop/LoopInLoop5.java
public class LoopInLoop5 {
	public static void main(String[] args) throws Exception {
		System.out.print("\n");
		for (int vert = 1; vert <= 8; vert++) {
			// Nested Loops gave me anxiety...
			for (int horz = vert; horz <= 8; horz++) {
				System.out.print(horz + " ");
			}
			System.out.print("\n");
		}
	}
}
